src/components/shared/nextButton.tsx

```diff
@@ -1,7 +1,12 @@
 import '../../styles/nextButton.scss';
 import {Link, useLocation} from 'react-router-dom';
 
-function NextButton(): JSX.Element {
+interface NextButtonProps {
+  isEnabled: boolean;
+  disable: () => void;
```

Comment:
rather than passing in `disable`, i'd recommend passing in an `onClick` to make this component more reusable